modules/forum/src/main/ForumPostApi.scala

```diff
@@ -223,14 +223,11 @@ final class ForumPostApi(
     postRepo.coll.update
       .one($id(post.id), post.erase)
       .void
-      .andDo:
-        Bus.pub(BusForum.ErasePost(post.id))
 
   def eraseFromSearchIndex(user: User): Funit =
     postRepo.coll
       .distinctEasy[ForumPostId, List]("_id", $doc("userId" -> user.id), _.sec)
-      .map: ids =>
-        Bus.pub(BusForum.ErasePosts(ids))
+      .map(ids => Bus.pub(BusForum.ErasePosts(ids)))
```

Comment:
the entire function should be gone

Reply:
How, I think We don't delete user's forum posts in db. Only remove forum post's indexes in elastic search. That's why I keep this.